src/Admin/AdminInterface.php

```diff
@@ -39,31 +40,32 @@
 /**
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  *
- * @method array                           configureActionButtons(string $action, ?object $object = null)
- * @method string                          getSearchResultLink(object $object)
- * @method void                            showMosaicButton(bool $isShown)
- * @method bool                            isDefaultFilter(string $name)                                         // NEXT_MAJOR: Remove this
- * @method bool                            isCurrentRoute(string $name, ?string $adminCode)
- * @method bool                            canAccessObject(string $action, object $object)
- * @method mixed                           getPersistentParameter(string $name)
- * @method array                           getExportFields()
- * @method array                           getSubClasses()
- * @method AdminInterface                  getRoot()
- * @method string                          getRootCode()
- * @method array                           getActionButtons(string $action, ?object $object)
- * @method FieldDescriptionCollection|null getList()
- * @method void                            setFilterPersister(?FilterPersisterInterface $filterPersister = null)
- * @method string                          getBaseRoutePattern()
- * @method string                          getBaseRouteName()
- * @method ItemInterface                   getSideMenu(string $action, ?AdminInterface $childAdmin = null)
- * @method void                            addParentAssociationMapping(string $code, string $value)
- * @method RouteGeneratorInterface         getRouteGenerator()
- * @method string                          getClassnameLabel()
- * @method AdminInterface|null             getCurrentChildAdmin()
- * @method string|null                     getParentAssociationMapping()
- * @method void                            reorderFormGroup(string $group, array $keys)
- * @method void                            defineFormBuilder(FormBuilderInterface $formBuilder)
- * @method string                          getPagerType()
+ * @method array                            configureActionButtons(string $action, ?object $object = null)
+ * @method string                           getSearchResultLink(object $object)
+ * @method void                             showMosaicButton(bool $isShown)
+ * @method bool                             isDefaultFilter(string $name)                                         // NEXT_MAJOR: Remove this
```

Comment:
Not related to this PR, but we can remove this line

Reply:
To many change on this PR. Lets do it in another PR. Mybe we should create "Clean up" PR? :sweat_smile: